### DISCUSSION TITLE
question about worker's task fetch behavior

### DISCUSSION THREAD

**Discussion by horoyoii** [celery/celery#7977] [Q&A]
Is there any way for a worker to patch only one task?

By default, if there is a ready task in the rabbitmq queue, the worker seems to pick it up even if other tasks are processing.

I only want to fetch one task at a time and when that task completes, I want to fetch another task then.

**Comment by thedrow** [comment]
You should set the prefetch multiplier to 1.
See [docs](https://docs.celeryq.dev/en/latest/userguide/configuration.html#std-setting-worker_prefetch_multiplier) for reference.
